sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java

```diff
@@ -39,56 +40,82 @@ public static void main(String[] args) {
         // Build a new SmsClient
         SmsClient smsClient = smsClientBuilder.buildClient();
 
```

Comment:
Can we remove main and convert this to a createSmsClient method.. also the code around smsOptions is no longer needed in this